templates/slurm_csv.submit.in

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+#SBATCH --job-name=%DETECTOR_VERSION%/%DETECTOR_CONFIG%/%CSV_FILE%
+#SBATCH --error=LOG/SLURM/grex_%A/grex_%A_%4a.err
+#SBATCH --output=LOG/SLURM/grex_%A/grex_%A_%4a.out
+#SBATCH --mail-type=FAIL,TIME_LIMIT,TIME_LIMIT_90
+#SBATCH --cpus-per-task=1
+#SBATCH --mem=3G
```

Comment:
Missing compared to osg_csv.submit: disk space 5G, since not supported on slurm (there is tmp dir space, but that's different apparently).